pkg/internal/templates/cloud-init.sh.template

```diff
@@ -44,6 +44,15 @@ mkdir -p '{{ $unit.DropIns.Path }}'
 {{- end -}}
 {{- end }}
 
+# Change existing worker to use docker registry-mirror
+file="/etc/docker/daemon.json"
+if [ $(jq -r 'has("registry-mirrors")' "${file}") == "false" ]; then
```

Comment:
are we sure jq is installed in our images ?

Reply:
for sure: https://github.com/metal-stack/metal-images/blob/master/debian/Dockerfile#L65